tsconfig.json

```diff
@@ -1,11 +1,8 @@
 {
-  "compilerOptions": {
-    "module": "commonjs",
-    "target": "es5",
-    "outDir": "./lib/cjs",
-    "declaration": true,
-    "noEmitOnError": true,
-    "noImplicitAny": true
-  },
-  "include": ["src/**/*"]
+    "compilerOptions": {
+        "lib": ["es2015", "dom"],
+        "noImplicitAny": true,
+        "noEmit": true
```

Comment:
Oh is this why you don't need `--noEmit` anymore?

Reply:
:nod: